bqschema_suite_test.go

```diff
@@ -1,10 +1,10 @@
-package bqschema_test
+package bqschema
```

Comment:
In this case it does not change how the program acts or tests, I generally just prefer to have tests keep as packaged with the _test to make testing only call the Public functions. I feel it helps generate a more accurate test coverage number.  All of that is opinion not based on anything besides my feelings.  If you could change it to the _test package name or give a better reason than my I like it as _test package name, I will merge after that reason or change.  Thanks.